src/openwrt/view/admin_thread/thread_setting.htm

```diff
@@ -160,23 +160,13 @@ <h3><%:Network Configuration%></h3>
 			</div>
 		</div>
 		<div class="cbi-value">
-			<% if state == 'disabled' then %>
-			<label class="cbi-value-title" style="margin-right:5%;">LeaderPartitionId</label>
+			<label class="cbi-value-title" style="margin-right:5%;">PartitionId</label>
 			<div class="cbi-value-title">
-			<input type="text" name="leaderpartitionid" value="<%=threadget("leaderpartitionid").Leaderpartitionid%>" style="width:50%;"/>
-			</div>
-			<div style="margin-left:30%;margin-top:1%;">
-				<span><img src="<%=resource .. "/cbi/help.gif"%>"/><%:leaderpartitionid must be a number%></span>
-			</div>
-			<% else %>
-			<label class="cbi-value-title" style="margin-right:5%;">LeaderPartitionId</label>
-			<div class="cbi-value-title">
-			<input type="text" name="leaderpartitionid" value="<%=threadget("leaderpartitionid").Leaderpartitionid%>" readonly="readonly" style="width:50%;"/>
+			<input type="text" name="partitionid" value="<%=threadget("partitionid").Partitionid%>" readonly="readonly" style="width:50%;"/>
```

Comment:
```suggestion
			<%=threadget("partitionid").Partitionid%>
```

Reply:
should be fine to use a readonly text?

Reply:
it will still be submitted even it's readonly.

Reply:
Should be fine if we don't use `partitionid` arguments?

Reply:
but why submit something not to be used?